Gemfile

```diff
@@ -11,7 +11,7 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem 'voxpupuli-test', '>= 1.0.0',  :require => false
```

Comment:
Is is related to this change ?

Reply:
this is cherry-picked from modulesync and required to make rspec-puppet-facts work